fmriprep/workflows/base.py

```diff
@@ -29,6 +32,11 @@ def base_workflow_enumerator(subject_list, task_id, settings):
         generated_workflow = base_workflow_generator(subject, task_id=task_id,
                                                      settings=settings)
         if generated_workflow:
+            generated_workflow.config['execution']['crashdump_dir'] =(
+                os.path.join(settings['output_dir'], 'log', subject)
```

Comment:
I'll have it make a directory with subject + timestamp, then in run reports have it look for the newest directory.